python/graphstorm/config/argument.py

```diff
@@ -1564,8 +1566,9 @@ def save_prediction_path(self):
             return self._save_prediction_path
 
         # if save_prediction_path is not specified in inference
-        # use save_embed_path
-        return self.save_embed_path
+        logging.warning("save_prediction_path is not set. Prediction results" \
```

Comment:
i think we shouldn't print this warning message. we usually print warning messages for problems. but not setting prediction path is a normal thing to do.